### ISSUE TITLE
Add Bluesky icon

### ISSUE THREAD

**Issue by BMariusS** [primefaces/primeicons#1176]
With Bluesky open for public it would be nice to have the icon added.

There is already a fontawesome equivalent here https://fontawesome.com/icons/bluesky?f=brands&s=solid

**Comment by L422Y** [comment]
really need this... is this repository abandoned? it's been almost a year without any changes... should we switch to a different library?

**Comment by onursenture** [comment]
> really need this... is this repository abandoned? it's been almost a year without any changes... should we switch to a different library?

No, it's not abandoned. We plan to release a new version soon. Thanks for your feedback.

**Comment by MrJibus** [comment]
Any news on a new version ? I also need the Bluesky icon :)
